packages/odyssey-storybook/src/utilities/Utilities.stories.mdx

```diff
@@ -0,0 +1,202 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Utilities/Utilities" />
+
+# Utilities
+
+## Box VS Text
```

Comment:
I'm not sure this is the right way to present the docs - these two aren' in opposition

Reply:
Agreed, this phrasing is just for us while we align. Each of these will have individual docs, likely only mentioning the other as a reference item.